blue-note/templates/page.html

```diff
@@ -1,10 +1,10 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:group {"tagName":"main","layout":{"type":"constrained"}} -->
-<main class="wp-block-group">
+<!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"bottom":"var:preset|spacing|100"}}},"layout":{"type":"constrained","wideSize":""}} -->
+<main class="wp-block-group" style="padding-bottom:var(--wp--preset--spacing--100)">
 
-	<!-- wp:post-title /-->
-	<!-- wp:post-content /-->
+	<!-- wp:post-title {"style":{"spacing":{"margin":{"bottom":"var:preset|spacing|100"}}},"fontSize":"huge"} /-->
```

Comment:
The font size might be better in theme.json.

Reply:
the post title has different sizes all over the theme, if we change it for this template we'll have to change other templates to compensate for that change anyway